unit-7/chat/index.html

```diff
@@ -0,0 +1,210 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Messenger Chat</title>
+    <link rel="stylesheet" href="style.css">
+    <script src='https://kit.fontawesome.com/a076d05399.js'></script>
+</head>
+<body>
+    <div class="wrap">
+        <div class="wrap__col-4">
+            <table>
+                <tr class="tr__search">
+                    <div class="wrap__col-4__search">
+                        <form action="" method="get">
+                            <input type="text" placeholder="Search...">
+                            <button class="search" type="submit"><i class="fa fa-search"></i></button>
+                        </form>
+                    </div>
+                </tr>
+
+                <tr>
+                    <div class="wrap__col-4__acc">
+                        <img class="avatar" src="./avatars/tracy-carpenter .jpg" alt="">
+                        <div class="wrap__col-4__acc--user">
+                            <p class="name">aiden chavez</p>
+                            <p class="status"><i class='fas fa-circle online'></i>online</p>
```

Comment:
```suggestion
                            <p class="status"><i class="fas fa-circle online"></i>online</p>
```
Trong HTML dùng nháy kép